### PR TITLE
#328 layout reorganization of user profile top section

### PR DESCRIPTION
**Pull request by Yog**
this is initial commit of better user profile layout, but it might be good to merge it, so we avoid conflicts in near future

### CODE REVIEW

File: volontulo/templates/common/badges.html [hunk 1]
```diff
@@ -1,12 +1,11 @@
 {% if badges %}
-    <h2>Odznaki użytkownika</h2>
-    <ul class="badges">
+    <ul class="badges list-unstyled">
         {% for badge in badges %}
             <li>
-                <span class="name priority-{{ badge.badge__priority }}">{{ badge.badge__name }}</span> x <span class="badge">{{ badge.badges }}</span>
+                <b class="name priority-{{ badge.badge__priority }}">{{ badge.badge__name }}</b> <span class="badge">{{ badge.badges }}</span>
```
Comment (filipgorczynski) [inline volontulo/templates/common/badges.html:5]: My suggestion is to not using representative elements: b, i, u. It's legacy after firsts HTML stsandards. strong would be better for this.

File: volontulo/templates/users/user_profile.html [hunk 2]
```diff
@@ -4,59 +4,83 @@
...
+            <img class="img-responsive" src="{{ MEDIA_URL }}{{ userprofile.get_avatar.0.image }}" alt="{{ user.username }}" />
+        </div>
+        <div class="media-body media-top">
+            <h2 class="lead">Użytkownik: {{ user.username }}</h2>
+            <ul class="list-unstyled">
+                <li><b>Real User ID:</b> <span class="badge">{{ user.id }}</span></li>
+                <li><b>UserProfile ID:</b> <span class="badge">{{ user.userprofile.id }}</span></li>
+            </ul>
+        </div>
+        <div class="media-body media-top">
```
Comment (filipgorczynski) [inline volontulo/templates/users/user_profile.html:18]: What do you think about moving this HTML wraps into common/badges.html? That will allow us to remove badges without leaving empty div elements.

File: volontulo/templates/common/badges.html (hunk 1 above)
Comment (Yog) [inline volontulo/templates/common/badges.html:5]: but didn't we agree earlier that `b` will stand for medium weight and `strong' will remain bold? 'D

File: volontulo/templates/users/user_profile.html [hunk 3]
```diff
@@ -4,59 +4,83 @@
...
+            {% endif %}
+            {% include 'users/my_offers.html' with offers=offers%}
         </div>
     </div>
 
     <div class="panel panel-default">
-        <div class="panel-heading lead">Galeria organizacji</div>
+        <h3 class="panel-heading lead">
+            <span class="glyphicon glyphicon-home" aria-hidden="true"></span>
+            Działania w instytucji czy organizacji
```
Comment (filipgorczynski) [inline volontulo/templates/users/user_profile.html:74]: This "czy" sounds strange. Maybe instytucji/organizacji?

File: volontulo/templates/users/user_profile.html (hunk 2 above)
Comment (Yog) [inline volontulo/templates/users/user_profile.html:18]: yeah, but the problem was that we can't wrap header element into proper header tags on diferent pages where we include that

**Pull request merged**